src/main/java/io/odpf/depot/bigquery/error/UnknownError.java

```diff
@@ -9,13 +9,17 @@
  * */
 public class UnknownError implements ErrorDescriptor {
 
+    private String reason;
+
+    private String message;
+
     @Override
     public boolean matches() {
         return false;
     }
 
     @Override
     public String toString() {
-        return "UnknownError";
+        return String.format("%s: %s", reason != null ? reason : "UnknownError", message != null ? message : "");
```

Comment:
What is the difference between `reason` and `message` fields ? I mean, what do each of these signify about the actual error ?

Also,  in case, both reason and message are empty strings (""), it'll print something like `:`, which might look weird.

Reply:
`reason` and `message` are the contract for the BigqueryError from InsertAllResponse. 
Based on the contract reason should not be null. `reason` only tells you what caused the error and the `message` is a more detailed description of the error. 

have updated for handling the empty string